pkg/serverless/daemon/daemon_test.go

```diff
@@ -19,18 +20,24 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/trace/testutil"
 )
 
+func TestMain(m *testing.M) {
+	origShutdownDelay := ShutdownDelay
+	ShutdownDelay = 0
```

Comment:
When calling `defer d.Stop()`, we do not actually care if data is flushed correctly, we're just doing it to clean up after ourselves.